incubator/mssql-linux/README.md

```diff
@@ -0,0 +1,126 @@
+1.  Open a terminal session/window
+2.  Type in the following
+```console
+$ echo -n "<MyNewStrongSaPassword>"| base64
+$ <base 64 output>
+```
+3.  Copy the Base64 output in step 2 into the templates/secret.yaml in the `password` field
+> Note - The `echo -n` in important to ensure no new lines are added at the end of the password
+
+You can install the chart with the release name `mymssql` as below.
+```console
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install --name mymssql incubator/mssql --set ACCEPT_EULA.value=Y
+```
+> Note - If you do not specify a name, helm will select a name for you.
+
+### Installed Components
+You can use `kubectl get` to view all of the installed components.
+
+```console{%raw}
+$ kubectl get all -l app=mssql
+NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/mymssql-mssql          1         1         1            1           9m
+
+NAME                           DESIRED   CURRENT   READY     AGE
+rs/mymssql-mssql-8688756468   1         1         1         9m
+
+NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/mymssql-mssql          1         1         1            1           9m
+
+NAME                           DESIRED   CURRENT   READY     AGE
+rs/mymssql-mssql-8688756468   1         1         1         9m
+
+NAME                                 READY     STATUS    RESTARTS   AGE
+po/mymssql-mssql-8688756468-x758g   1/1       Running   0          9m
+
+NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+svc/mymssql-mssql   ClusterIP   10.104.152.61   <none>        1433/TCP   9m
+
+```
+
+## Connecting to SQL Server Instance
+1.  Run the following command
+```console
+$ kubectl run dbatoolbox --image=microsoft/mssql-tools -ti --restart=Never --rm=true -- /bin/bash
+$ sqlcmd -S {{ template "mssql.fullname" . }}.{{ .Release.Namespace }} -U sa
+Password: <Enter SA Password>
+```
+
+## Values
+The configuration parameters in this section control the resources requested and utilized by the ZooKeeper ensemble.
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| ACCEPT_EULA | EULA that needs to be accepted.  It will need to be changed via commandline or values.yaml. | N |
+| edition | The edition of SQL Server to install | Express |
+| image | |  |
+| repository | The docker hub repo for SQL Server | microsoft/mssql-server-linux |
+| tag | The tag for the image | latest |
+| pullPolicy | The pull policy for the deployment | IfNotPresent |
+| service | | |
+| name | Service Name | mssqlsrvr |
+| type | Service Type | ClusterIP |
+| externalPort | External Port | 1433 |
+| internalPort | Internal Port | 1433 |
+| livenessprobe | | |
```

Comment:
Either liveness and readiness probes should be separate tables, or the parameters should be described as `livenessprobe.initialDelaySeconds` and so on.

Reply:
Fixed.  I put this in it's own table(s)